src/Model/CampaignDraft.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Mailjet\MailjetBundle\Model;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaigndraft/
+ */
+class CampaignDraft {
+
+    const LOCALE_KEY = 'Locale';
+    const SENDER_KEY = 'Sender';
+    const SENDEREMAIL_KEY = 'SenderEmail';
+    const SUBJECT_KEY = 'Subject';
+    const CONTACTLISTID_KEY = 'ContactsListID';
+
+    //Mandatory properties (must be set through constructor)
+    protected $locale;
+    protected $sender;
+    protected $senderEmail;
+    protected $subject;
+    protected $contactsListID;
+    //Optional properties (array)
+    protected $optionalProperties = null;
+    //content
+    protected $content = null;
+    //id
+    protected $id = null;
+
+    /**
+     * @param $Locale
+     * @param $Sender
+     * @param $SenderEmail
+     * @param $Subject
+     * @param $ContactsListID
+     * @param optionalProperties
+     */
+    public function __construct($locale, $sender, $senderEmail, $subject, $contactsListID, $optionalProperties = null) {
```

Comment:
This is a Model layer, to me it makes sense to keep it this way. If we are not going to build anything ontop of the wrapper and keep it as "simple" as the php wrapper, I do not see a purpose of having the Model layer at all :) .